src/Razor/Directory.Build.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <!-- Only include .NET Framework files when building for .NET Framework -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net472'">
+    <Compile Remove="**\*.NetFX.cs" />
+    <Compile Remove="**\NetFx\**\*.*" />
+    <Compile Remove="**\*_NetFx\**\*.*" />
+
+    <None Include="**\*.NetFX.cs" />
+    <None Include="**\NetFx\**\*.*" />
+    <None Include="**\*_NetFx\**\*.*" />
+  </ItemGroup>
+
+  <Import Project="$(RepoRoot)Directory.Build.targets" />
```

Comment:
Nit: `<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />`

That way if someone adds a `Directory.Build.targets` file to the `src` folder, they will be less likely to call for your head :)